shared/common-adapters/avatar.desktop.js

```diff
@@ -139,7 +139,7 @@ class Avatar extends Component<void, Props, State> {
         {this.props.backgroundColor &&
           <div
             style={{...avatarStyle,
-              ...borderStyle,
```

Comment:
Max and I chatted live about this - the borderColor attribute is used in the nav bar so we're keeping it.